csrc/python_frontend/python_bindings.cpp

```diff
@@ -677,48 +678,91 @@ void initNvFuserPythonBindings(PyObject* module) {
   heuristic_params.def(
       "__repr__", [](const HeuristicParams& self) { return self.toString(); });
 
+#define INITPARAMS(cfg, internal_type)                                     \
```

Comment:
Should we define all macros upfront?
1. INITPARAMS
2. PARAM
3. TILEPARAM
4. CBPARAM

Reply:
`TILEPARAM` and `CBPARAM` are specific to matmul, so I now define those at the top of the MatmulParams section, as well as the `py::class_` and `py::enum_` definitions. This way you can easily read out the `matmul_config` definition.